views/admin/tpl/transaction_tab.tpl

```diff
@@ -0,0 +1,52 @@
+[{include file="headitem.tpl" title="GENERAL_ADMIN_TITLE"|oxmultilangassign}]
+
+<script type="text/javascript">
+<!--
+function copyToClipboard(text)
```

Comment:
Please, remove this commented out code in case it is not needed anymore.

Reply:
This is not commented out – Javascript is still executed for browsers that understand it. Oxid uses this outdated pattern for browser compatibility: https://stackoverflow.com/questions/808816/are-html-comments-inside-script-tags-a-best-practice